src/epics/mutate.js

```diff
@@ -37,65 +29,16 @@ import { getDeprecatedSuccessPayload } from '../util/fetchUtils';
  * @module MutateEpic
  */
 
-/**
- * get a function that receives a mutation-specific action and returns the
- * corresponding fetch Promise
- *
- * The wrapping function needs current app state in order to apply correct
- * config
- *
- * @param {String} method the desirect HTTP method ('post', 'delete', 'patch')
- * @param {Object} { apiUrl } from state.config
- * @param {Object} postAction, providing query, onSuccess, and onError
- * @return {Promise} results of the fetch, either onSuccess or onError
- */
-const getMethodQueryFetch = (method, fetchQueries, store) => query => {
-	// force presence of meta.method
-	query.meta = { ...(query.meta || {}), method };
-	const { config: { apiUrl } } = store.getState();
-	return fetchQueries(apiUrl)([query]);
-};
-
-/**
- * Make the mutation call to the API and send the responses to the appropriate
- * places
- *
- * 1. Always send successful mutation responses to API_SUCCESS action
- * 2. If mutation action has an 'onSuccess' action creators, send successful
- *    responses there as well
- * 3. Failed mutation responses will be sent to the mutation action's `onError`
- */
-const doFetch$ = fetchQuery => ({ query, onSuccess, onError }) =>
-	Observable.fromPromise(fetchQuery(query)) // make the fetch call
-		.mergeMap(({ successes, errors }) => {
-			const responses = getDeprecatedSuccessPayload(successes, errors);
-			const actions = [
-				...successes.map(api.success), // send the successes to success
-				...errors.map(api.error), // send errors to error
-				apiSuccess(responses),
-			];
-			if (onSuccess) {
-				actions.push(onSuccess(responses));
-			}
-			return Observable.from(actions);
-		})
-		.catch(err => {
-			const actions = [api.fail(err), apiError(err)];
-			if (onError) {
-				actions.push(onError(err));
-			}
-			return Observable.from(actions);
-		});
-
-const getMethodEpic = method => fetchQueries => (action$, store) =>
+const getMethodEpic = method => action$ =>
```

Comment:
done - this whole module is deprecated so hopefully no one will actually ever have to _read_ the docs, though :)